src/FormBuilder/Actions/DequeueAdminScriptsInFormBuilder.php

```diff
@@ -22,7 +22,7 @@ public function __invoke()
         if ($this->isFormBuilderRoute()) {
             $wpScripts = wp_scripts();
 
-            $wpScriptsRegistered = array_column($wpScripts->registered, 'handle');
+            $wpScriptsRegistered = []; //array_column($wpScripts->registered, 'handle');
```

Comment:
We need some core scripts for the Media Library integration to work. This needs to be revisited so we don't just enqueue everything.